XXXXX-template.md

```diff
@@ -1,85 +1,262 @@
 ---
-number: 'XXXXX' (update this part after PR creation)
+number: 'XXXXX'
 title: Title of SIP
-authors: First Author's Name (author1@example.org), Second Author's name (author2@example.org)
+authors: 
+  - First Author's Name (author1@example.org)
+  - Second Author's name (author2@example.org)
 sponsors: First Sponsor's Name (expert@example.org)
-created: (fill me in with today's date, YYYY-MM-DD)
-type: Standard/Ecosystem/Meta
+created: 2025-06-27
+type: Standard
```

Comment:
If this is still a template, it should have the 3 options.
If you want this proposal to be a Meta IP, then copy the template and modify that file, not the template directly